### PR TITLE
Include median in master

### PR DESCRIPTION
**Pull request by lrubiorod**

### CODE REVIEW

**Comment by tmpolaczyk** [comment]
This was already reviewed in #2049

File: config/src/loaders/toml.rs [hunk 1]
```diff
@@ -234,18 +234,16 @@ enabled = false
...
         )
         .unwrap();
 
         assert_eq!(empty_config.tapi, Tapi::default());
-        assert!(!empty_config.tapi.oppose_wip0014);
-        assert!(!empty_config.tapi.oppose_wip0016);
-        assert!(!config_oppose_0016.tapi.oppose_wip0014);
-        assert!(config_oppose_0016.tapi.oppose_wip0016);
+        assert!(!empty_config.tapi.oppose_wip0017);
+        assert!(config_oppose_0017.tapi.oppose_wip0017);
```
Comment (tmpolaczyk) [inline config/src/loaders/toml.rs:247]: This test was also used to check that unset fields default to false. Can you add something like that again?

        assert!(!empty_config.tapi.oppose_wip0018);
        assert!(!config_oppose_0017.tapi.oppose_wip0018);

**Comment by lrubiorod** [comment]
> This was already reviewed in #2049

So... approved!! :P

File: data_structures/src/mainnet_validations.rs [hunk 2]
```diff
@@ -644,24 +665,24 @@ mod tests {
...
         let (epoch, old_wips) = t.initialize_wip_information(Environment::Mainnet);
-        // The first block whose vote must be counted is the one from WIP0014
-        let init_epoch_wip0014 = 522240;
-        assert_eq!(epoch, init_epoch_wip0014);
+        // The first block whose vote must be counted is the one from WIP0017-0018-0019
+        let init_epoch_wip0017_18_19 = 656640;
+        assert_eq!(epoch, init_epoch_wip0017_18_19);
         // The TapiEngine was just created, there list of old_wips must be empty
         assert_eq!(old_wips, HashSet::new());
         // The list of active WIPs only contains the first, second, and third hard forks
```
Comment (tmpolaczyk) [inline data_structures/src/mainnet_validations.rs:673]:
```suggestion
        // The list of active WIPs should match those defined in `wip_info`
```

File: rad/src/script.rs [hunk 3]
```diff
@@ -211,10 +213,17 @@ fn errorify(kind: RadError) -> RadError {
...
+    active_wips: Option<&ActiveWips>,
 ) -> Result<Vec<RadonCall>, RadError> {
     let unknown_filter = |code| RadError::UnknownFilter { code };
     let unknown_reducer = |code| RadError::UnknownReducer { code };
 
+    let active_wips = if let Some(active_wips) = active_wips {
+        active_wips.clone()
+    } else {
+        current_active_wips()
+    };
```
Comment (tmpolaczyk) [inline rad/src/script.rs:225]: I think this clone can be avoided, by doing something like this:

```suggestion
    let current_wips;
    let active_wips = if let Some(active_wips) = active_wips {
        active_wips
    } else {
        current_wips = current_active_wips();
        &current_wips
    };
```

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**